### PR TITLE
feature/update contact screen

### PR DESCRIPTION
**Pull request by dtngoc99**
## Change Description 👋

Update contact screen

- Change contact information

## Demo 📺

https://loom.com/share/b3fd8559709f48d094becc86c250c1b4

### CODE REVIEW

**Comment by dtngoc99** [comment]
image

**Comment by dtngoc99** [comment]
image

File: src/contacts/components/ContactForm.vue [hunk 1]
```diff
@@ -169,43 +191,75 @@ export default {
...
       SUPPLIER_TYPE: SUPPLIER_TYPE,
       BOTH_TYPE: BOTH_TYPE,
-      countries: [
-        { name: "Vietnam", value: "+84", color: "#f2b334" },
-        { name: "English", value: "+965", color: "#f2b334" },
-      ],
+      isDialog: false,
+      dialogTitle: "",
+      dialogContent: "",
+      selectedContact: undefined,
```
Comment (luantc13) [inline src/contacts/components/ContactForm.vue:206]: We should also remove `selectedContact` here.

File: src/contacts/components/ContactForm.vue [hunk 2]
```diff
@@ -169,43 +191,75 @@ export default {
...
+      contactInfor: {
+        id: null,
+        firstName: "",
+        lastName: "",
+        email: "",
+        phoneNumber: "",
+        address: "",
+        contactType: 2,
```
Comment (luantc13) [inline src/contacts/components/ContactForm.vue:214]: Using constant `CUSTOMER_TYPE`, `SUPPLIER_TYPE` and `BOTH_TYPE` is a better choice.

**Review by luantc13 — APPROVED** [review]
Great work!
No major issues here, happy to merge.

**Review by truonghungit — APPROVED** [review]

**Pull request merged**